.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: false
 cache: "bundler"
 matrix:
   include:
-    - rvm: "2.6"
+    - rvm: "2.7"
```

Comment:
Addressed in https://github.com/litaio/lita/commit/4434336101213502c90c260d67385fccbbc4708c#diff-354f30a63fb0907d4ad57269548329e3R7